crates/bevy_window/src/window.rs

```diff
@@ -989,6 +989,11 @@ pub enum WindowMode {
     /// When setting this, the window's physical size will be modified to match the size
     /// of the current monitor resolution, and the logical size will follow based
     /// on the scale factor, see [`WindowResolution`].
+    ///
+    /// Note: As this mode respects the scale factor provided by the operating system,
+    /// the window's logical size may be different from its physical size.
+    /// If you want to avoid that behavior, you can use the [`WindowResolution::set_scale_factor_override`]
+    /// function to set the scale factor to 1.0.
```

Comment:
This definitely seems like the right place to add this detail to the docs. I think though that we should also mention the `with_scale_factor_override` builder method as it works better with the `DefaultPlugins::set` builder interface that most users will probably be using. 

```suggestion
    /// If you want to avoid that behavior, you can use the [`WindowResolution::set_scale_factor_override`] function
    /// or the [`WindowResolution::with_scale_factor_override`] builder method to set the scale factor to `1.`.
```